build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -6,62 +6,84 @@
 """Defines the groups of models."""
 
 from e2e_test_framework.models import matmul, tflite_models, torch_models, tf_models
+from e2e_test_framework.definitions.common_definitions import CpuBenchmarkConfig
```

Comment:
nit: We usually import the module instead of classes in it (https://google.github.io/styleguide/pyguide.html#22-imports).

Also the imports are sorted lexicographically on the module path after `from`.

```suggestion
from e2e_test_framework.definitions.common_definitions
from e2e_test_framework.models import matmul, tflite_models, torch_models, tf_models
```